kustomize/base/all-in-one/resources/deploy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
```

Comment:
Looks like we're missing:

- `Service` for the `Deployment`
- `Route` for the API on the Manager
- `Route` for the Ingress

Reply:
I've included the `Route` in the `prod` overlay. In that way we keep the `base` not OCP specific: we can create a `dev` or `minikube` overlay that deploys the stuff on a minikube cluster. Let me know if it's ok for you @robpblake

Reply:
Sounds good 👍